e2e/benchmarks/local-benchmark/index.html

```diff
@@ -496,6 +504,8 @@ <h2>TensorFlow.js Model Benchmark</h2>
         state.numProfiles = Number(urlState.get('profile'));
       if (urlState.has('benchmark'))
         state.benchmark = urlState.get('benchmark');
+      if (urlState.has('modelUrl'))
+        state.modelUrl = urlState.get('modelUrl');
```

Comment:
This is probably okay for testing and benchmarking, but keep in mind [it may fail if the model url has characters like `?` or `&`](https://stackoverflow.com/questions/1737935/what-is-the-recommended-way-to-pass-urls-as-url-parameters). It would be better to encode the URL with `encodeURIComponent`, but the format makes it hard for users to create new URLs.

It looks like you're hooking into the UI element that allows setting the custom model URL. If we can make this bi-directional (i.e. if you set the url in the UI, it appears as a URL parameter), then we can use the correct encoding, and users can input new URLs in the UI.